### PR TITLE
Odst 260 fs org creation update page page

### PR DESCRIPTION
**Pull request by Andydawg313**
# To be Completed by the Pull Requester

### Checklist:

- [x] I have performed a self-review of my code to ensure that unintended code is not introduced.
- [x] I have commented my code, particularly in hard-to-understand areas to ensure that the next group of developers can understand my code.
- [x] I have added tests (if appropriate) that prove my fix/feature functions so that future modifications to the code do not break my additions.

### Description:

Created and add tests for Org creation and edit pages

### Type of change (Pick Any)

2. New feature. Non-breaking change which adds functionality, example: new page on the frontend.

### How Has This Been Tested?

Wrote FE, BE and E2E tests

---

# To be Completed By Reviewers

Reviewer # 01

- [x] I have performed a review of the code and checked for possible unintentional edits.
- [x] I concur with the amount of comments made.
- [x] I concur with the amount of tests made.

Reviewer # 02

- [x] I have performed a review of the code and checked for possible unintentional edits.
- [x] I concur with the amount of comments made.
- [x] I concur with the amount of tests made.

### CODE REVIEW

File: apps/ods-api/schema.prisma [hunk 1]
```diff
@@ -143,3 +143,9 @@ enum OrgTier {
   SQUADRON
   OTHER
 }
+
+// model OrgTiers {
```
Comment (BrendenMcGrew) [inline apps/ods-api/schema.prisma]: Did you mean to leave this in?

File: apps/ods-api/src/org/org.service.ts [hunk 2]
```diff
@@ -1,11 +1,272 @@
-import { Injectable } from '@nestjs/common';
-import { Org, Prisma, User, Feedback } from '.prisma/ods/client';
+import {
+  BadRequestException,
+  Injectable,
+  NotImplementedException,
+} from '@nestjs/common';
+import { Org, Prisma, User, Feedback, OrgTier } from '.prisma/ods/client';
 import { PrismaService } from '../prisma/prisma.service';
+//import { FindManyOrgArgs, OrgWhereInput } from '@odst/types/ods';
```
Comment (BrendenMcGrew) [inline apps/ods-api/src/org/org.service.ts]: I'd get rid of this if there is no plan to use it

File: apps/ods/src/app/create-org/create-org.component.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        *ngIf="submitSuccess"
+        class="mt-3 bg-green-500 bg-opacity-50 border-2 border-green-400"
+      >
+        <mat-card-content
+          class="flex justify-center text-sm space-y-3 text-justify"
+        >
+          <p>Your Unit has been successfully created!</p>
+        </mat-card-content>
+      </mat-card>
+      <mat-card
```
Comment (BrendenMcGrew) [inline apps/ods/src/app/create-org/create-org.component.html:139]: I don't think this form of error is consistent with the rest of the program. Ideally a mat snack bar or mat error under the fields would make better sense if there was an issue submitting the organization.

File: apps/ods/src/app/create-org/create-org.component.ts [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  childrenConnection(selectedChildren: string[]): { name: string }[] {
+    const result: any[] = [];
+    for (let i = 0; i < selectedChildren.length; i++) {
+      result[i] = { name: selectedChildren[i] };
+    }
+    return result;
+  }
+
+  async submit() {
+    this.form.value['orgName'] = this.form.value['orgName']
```
Comment (BrendenMcGrew) [inline apps/ods/src/app/create-org/create-org.component.ts:151]: Could this possibly be a for each loop? Not really an issue, just a question. This won't need to be 'fixed' per say

File: apps/ods/src/app/edit-org/edit-org.component.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        *ngIf="submitSuccess"
+        class="mt-3 bg-green-500 bg-opacity-50 border-2 border-green-400"
+      >
+        <mat-card-content
+          class="flex justify-center text-sm space-y-3 text-justify"
+        >
+          <p>Successfully edited organization!</p>
+        </mat-card-content>
+      </mat-card>
+      <mat-card
```
Comment (BrendenMcGrew) [inline apps/ods/src/app/edit-org/edit-org.component.html:122]: Same issue as before with error handling/displaying being consistent

File: apps/ods/src/app/edit-org/edit-org.component.ts [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+      result[i] = { name: selectedChildren[i] };
+    }
+    return result;
+  }
+
+  async submit() {
+    this.form.value['confirmName'] = this.form.value['confirmName']
+      .trim()
+      .toUpperCase();
+    let temp = '';
```
Comment (BrendenMcGrew) [inline apps/ods/src/app/edit-org/edit-org.component.ts]: This is still labeled as temp, which I can assume means temporary. Is this intentional and if so, please rename it so I know what it means later.

File: apps/ods/src/app/create-org/create-org.component.ts (hunk 4 above)
Comment (Mikehickey7) [reply apps/ods/src/app/create-org/create-org.component.ts:151]: i think its fine how it is

File: apps/ods/src/app/edit-org/edit-org.component.ts (hunk 6 above)
Comment (Mikehickey7) [reply apps/ods/src/app/edit-org/edit-org.component.ts]: set to orgName instead, cause that is the org name being set depending on certain logic

File: apps/ods/src/app/create-org/create-org.component.html (hunk 3 above)
Comment (Andydawg313) [reply apps/ods/src/app/create-org/create-org.component.html:139]: we have decided that this is fine, it is in line with the figma and makes sense for what it is doing

**Review by pmeskridge — COMMENTED** [review]
response-lookup.service.spec.ts line 33 is failing when we do the pipeline test.  This might be due to the mock data made on another project.  Please check to see if line 33 should refer to 'Sandy' or ''.

**Review by REDRICK696 — APPROVED** [review]

**Review by pmeskridge — APPROVED** [review]

**Pull request merged**